src/devneya/cli.clj

```diff
@@ -50,9 +48,21 @@
 
 (defn run-cli [config args]
   (let [{:keys [prompt options exit-message ok?]} (validate-args args)]
-    (when exit-message
-      (err/exit (if ok? 0 1) exit-message)) 
-    (try
-      (prompt/make-initial-prompt (:OPENAI_KEY config) prompt (:output-filename options) (:REQUSET_LOG_PATH config))
-      (when (= (:exec options) true) (exec/exec-code config (:output-filename options)))
-      (catch Throwable e (println e)))))
\ No newline at end of file
+    (when exit-message (err/exit (if ok? 0 1) exit-message))
+    (if (= (:all options) true)
+      (try
+      (loop [i 0]
+        (if (= i 0)
+          (prompt/make-initial-prompt config prompt)
+          (prompt/make-fix-prompt config))
+        (exec/exec-code config)
+        (if (and (not= (slurp (:DENO_ERROR_FILENAME config)) "") (< i (:MAX_REPS config)))
```

Comment:
Why use (not= "" str) instead of (not-empty str)?

Reply:
My bad, i'll fix it.